caravel/templates/moderation/view.html

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.html" %}
+{% block title %}Moderate Inquiries{% endblock %}
+{% block head %}
+  {{ super() }}
+  <script src="/static/js/moderation.js"></script>
+{% endblock %}
+{% block content %}
+  <h2>Inquiries</h2>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>User Agent</th>
+        <th>IP</th>
+        <th>Email</th>
+        <th>Listing</th>
+        <th>Message</th>
+        <th>Moderation</th>
+      </tr>
+    </thead>
+    {% for inquiry in inquiries %}
+    <tr>
+      <td>{{ user_agent(inquiry.principal.device.user_agent) }}</td>
+      <td>{{ inquiry.principal.device.ip_address }}</td>
+      <td>{{ inquiry.principal.email }}</td>
+      <td><a href="{{ url_for('show_listing',
+            listing=inquiry.listing.get()) }}">
+          {{ inquiry.listing.get().title }}</td>
+      <td>{{ inquiry.message }}</td>
```

Comment:
Hmm, good point. Should we see how it works on in practice? This whole form feels a little janky — it might make a good candidate for a React.js rewrite.